CHANGELOG.md

```diff
@@ -6,6 +6,14 @@ All notable changes to [bpmn-js](https://github.com/bpmn-io/bpmn-js) are documen
 
 ___Note:__ Yet to be released changes appear here._
 
+## [14.0.1](https://github.com/bpmn-io/bpmn-js/compare/v14.0.0...v14.0.1) (2023-10-12)
+
+
+### Bug Fixes
```

Comment:
Let me know what you think.

Reply:
I like our existing template, because it is compact, and meaningful.

What you propose hides a major feature (cf. https://github.com/bpmn-io/bpmn-js/issues/557).

Reply:
Also this clearly shows a gap in `release-please`. A new feature (local spacetool) may never be a patch.

Reply:
> A new feature (local spacetool) may never be a patch.

I totally agree with you. That's why we need to be extra careful with commit messages if we want automation. So it's no more "oh you labeled is as patch-releasable, but it's a feature in fact". Or we review automation result manually.

Reply:
Perhaps it would make more sense to do sth like:

```
deps: update diagram-js to v12.2.0

feat: support local space tool

Closes #758
```

https://github.com/googleapis/release-please#what-if-my-pr-contains-multiple-fixes-or-features

Reply:
Let's discuss this. If we want automation, then we want to be super strict about our commits. If we think we can be, then let's try it :).

Reply:
Hmm custom changelog seems to be doable, but I couldn't find a way to do it in the github action:

https://github.com/googleapis/release-please/blob/main/docs/customizing.md#adding-additional-changelog-types

I did not dive deep though.